# >>>TODO: this needs tunning
waypoint_A: [-3.98119,-0.839,1.0]
waypoint_B: [-4.062751,-8.37063694,-1.57]
waypoint_C: [-2.452,-10.67,1.57]

target_pose: [0.537, -0.185, 0.561, 1.285, -1.494, 1.826]